src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,50 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<F, S> {
```

Comment:
According to `Type Parameter Naming Conventions` https://docs.oracle.com/javase/tutorial/java/generics/types.html it will be better to name `Pair<T, V>`